recipes/openimageio/3.x/conanfile.py

```diff
@@ -0,0 +1,303 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, copy, get, rm, rmdir
+from conan.tools.microsoft import is_msvc, is_msvc_static_runtime
+from conan.tools.scm import Version
+from conan.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=2.0.0"
+
+
+class OpenImageIOConan(ConanFile):
+    name = "openimageio"
+    description = (
+        "OpenImageIO is a library for reading and writing images, and a bunch "
+        "of related classes, utilities, and applications. There is a "
+        "particular emphasis on formats and functionality used in "
+        "professional, large-scale animation and visual effects work for film."
+    )
+    topics = ("vfx", "image", "picture")
+    license = "Apache-2.0", "BSD-3-Clause"
+    homepage = "http://www.openimageio.org/"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_dicom": [True, False],
+        "with_ffmpeg": [True, False],
+        "with_freetype": [True, False],
+        "with_giflib": [True, False],
+        "with_hdf5": [True, False],
+        "with_libheif": [True, False],
+        "with_libjpeg": ["libjpeg", "libjpeg-turbo"],
+        "with_libjxl": [True, False],
+        "with_libpng": [True, False],
+        "with_libwebp": [True, False],
+        "with_opencv": [True, False],
+        "with_openjpeg": [True, False],
+        "with_openvdb": [True, False],
+        "with_ptex": [True, False],
+        "with_raw": [True, False],
+        "with_tbb": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_dicom": False,  # Heavy dependency, disabled by default
+        "with_ffmpeg": True,
+        "with_freetype": True,
+        "with_giflib": True,
+        "with_hdf5": True,
+        "with_libheif": True,
+        "with_libjpeg": "libjpeg",
+        "with_libjxl": True,  # TODO: Currently produces link failues
+        "with_libpng": True,
+        "with_libwebp": True,
+        "with_openjpeg": True,
+        "with_openvdb": False,  # FIXME: broken on M1
```

Comment:
Is OpenImage's build broken, or is the breakage in openvdb?

Reply:
Good question. Didn't test, just copied from the 2.x (all) recipe. Will retry if it still fails.

Reply:
It looks like it works in the meantime, so changed it to true. Or maybe we better leave it at false to keep old behaviour but remove the comment only, what do you think?

Reply:
I think new major versions are ok with default changes of options, specially if that means following upstream decision to turn it on if available.